src/Apps/Order/Components/TransactionSummary.tsx

```diff
@@ -53,6 +56,10 @@ export class TransactionSummary extends React.Component<
       textOverflow: "ellipsis",
     } as any
 
+    const isMakeOfferFlow =
```

Comment:
General question throughout: Do we want to be referring to this as the `makeOfferFlow`? or just `offerFlow`? The "Make" seems a little redundant + tied to current business naming.

On the other hand it might be a nice distinction here to talk about the `makeOfferFlow` vs. the `counterofferFlow`. 🤷‍♀️ food for thought but interested in what others think.

Reply:
I think `offer` and `counteroffer` is how its used in everyday language, so that maps nicely.